mk/testcases/slackware-15.0

```diff
@@ -0,0 +1,11 @@
+NAME=Slackware
+VERSION="15.0"
+ID=slackware
+VERSION_ID=15.0
+PRETTY_NAME="Slackware 15.0 x86_64 (post 15.0 -current)"
+ANSI_COLOR="0;34"
+CPE_NAME="cpe:/o:slackware:slackware_linux:15.0"
+HOME_URL="http://slackware.com/"
+SUPPORT_URL="http://www.linuxquestions.org/questions/slackware-14/"
+BUG_REPORT_URL="http://www.linuxquestions.org/questions/slackware-14/"
+VERSION_CODENAME=current
```

Comment:
please add a new line for this file.

Reply:
@minli1 The new line at the end of `/etc/os-release` is inexistant on the dozens of systems  I have access to (Slackware, Debian, RockyLinux, freeBSD, etc).

Do you really need it ?

Reply:
that's fine. we could add them if needed.